templates/data/go-toolkit-validation.yaml

```diff
@@ -23,6 +23,16 @@ templates:
             pubsub_topic: "prowjobs"
             request_memory: 1Gi
             request_cpu: 100m
+          testing_required:
```

Comment:
I see that's the same as the `testing` config. Can we just use testing config and then in the job itself override the optional and set it to false ?

Reply:
I wanted to do it like that, but I noticed that this will remove the optional attribute from the rendered template, which I am not sure what does it mean, that's why I went for explicitly making it not optional.

Anyway, I did it again as you suggested.

Reply:
looks good now, let's check if the `optional` flag, which got removed in the templated file defaults to false.